db/warehouse/migrate/20240221195839_migrate_auto_exit_rules.rb

```diff
@@ -0,0 +1,36 @@
+#  Copyright 2016 - 2024 Green River Data Analysis, LLC
```

Comment:
Confirming, do we want to use the db migration file to do this data migration even though [some consider it an anti pattern](https://www.ombulabs.com/blog/rails/data-migrations/three-useful-data-migrations-patterns-in-rails.html)?

Related question, is it ok for this migration to cause any downtime from the auto-exit config UI's perspective? I guess what we might do if it were critical to prevent any downtime, is to modify the graphql code and the auto-exit config job code so that they can use both the old and the new model for awhile, then run the migration, then push an update again that removes the old model. I haven't done that here because @gigxz you mentioned that there is only 1 auto-exit config currently configured anywhere, so maybe this isn't where our effort should go?

Reply:
Yep this should be fine! Agree not worth the effort, we have one job that runs in the evening